vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -2003,42 +1644,31 @@ - (void)unblindedTokensReady
   }
 }
 
-- (void)deleteUnblindedTokensForPromotion:(const std::string&)promotion_id callback:(ledger::ResultCallback)callback
-{
-  // TODO please implement
-}
-
-- (void)getTransactionReport:(const ledger::ActivityMonth)month year:(const uint32_t)year callback:(ledger::GetTransactionReportCallback)callback
-{
-  // TODO please implement
-}
-
-- (void)getContributionReport:(const ledger::ActivityMonth)month year:(const uint32_t)year callback:(ledger::GetContributionReportCallback)callback
-{
-  // TODO please implement
-}
-
-- (void)getIncompleteContributions:(ledger::GetIncompleteContributionsCallback)callback
-{
-  // TODO please implement
-}
-
-- (void)getContributionInfo:(const std::string&)contribution_id callback:(ledger::GetContributionInfoCallback)callback
-{
-  // TODO please implement
-}
-
-- (void)updateContributionInfoStepAndCount:(const std::string&)contribution_id step:(const ledger::ContributionStep)step retry_count:(const int32_t)retry_count callback:(ledger::ResultCallback)callback
+- (void)reconcileStampReset
 {
   // TODO please implement
```

Comment:
Same with this? Seems these are newly added

Reply:
answered above